modules/azure/aks/variables.tf

```diff
@@ -204,3 +204,13 @@ variable "log_eventhub_authorization_rule_id" {
   description = "The authoritzation rule id for event hub"
   type        = string
 }
+
+variable "dns_zones" {
```

Comment:
might wanna call this `dns_zone` to keep the same naming in every module?

Reply:
The name for all other variables is wrong. Was planning on updating the other modules. This is a list of zones not a singe zone.

Reply:
True, that makes more sense